t/api/08-jobtemplates.t

```diff
@@ -1110,6 +1110,249 @@ subtest 'References' => sub {
+    - RAID1:
+        *raid1
+    - ext4_uefi-staging:
+        *ext4
+    - minimal+base
+    sle-12-SP5-Server-DVD-D-Staging-x86_64:
+    - gnome
+    - cryptlvm_minimal_x
+    - default_install:
+        *default_install
+    - installcheck
+    - migration_zdup_offline_sle12sp2_64bit-staging:
+        *zdup
+    - rescue_system_sle11sp4
+    - RAID1:
+        *raid1
+    - ext4_uefi-staging:
+        *ext4
+    - minimal+base
+    sle-12-SP5-Server-DVD-E-Staging-x86_64:
+    - cryptlvm
+    - gnome
+    - cryptlvm_minimal_x
+    - default_install:
+        *default_install
+    - installcheck
+    - migration_zdup_offline_sle12sp2_64bit-staging:
+        *zdup
+    - rescue_system_sle11sp4
+    - RAID1:
+        *raid1
+    - ext4_uefi-staging:
+        *ext4
+    - minimal+base
+    sle-12-SP5-Server-DVD-H-Staging-x86_64:
+    - gnome
+    - cryptlvm_minimal_x
+    - default_install:
+        *default_install
+    - installcheck
+    - migration_zdup_offline_sle12sp2_64bit-staging:
+        *zdup
+    - rescue_system_sle11sp4
+    - RAID1:
+        *raid1
+    - ext4_uefi-staging:
+        *ext4
+    - minimal+base
+    sle-12-SP5-Server-DVD-S-Staging-x86_64:
+    - gnome
+    - cryptlvm_minimal_x
+    - default_install:
+        *default_install
+    - installcheck
+    - migration_zdup_offline_sle12sp2_64bit-staging:
+        *zdup
+    - rescue_system_sle11sp4
+    - RAID1:
+        *raid1
+    - ext4_uefi-staging:
+        *ext4
+    - minimal+base
+    sle-12-SP5-Server-DVD-V-Staging-x86_64:
+    - gnome
+    - cryptlvm_minimal_x
+    - default_install:
+        *default_install
+    - installcheck
+    - migration_zdup_offline_sle12sp2_64bit-staging:
+        *zdup
+    - rescue_system_sle11sp4
+    - RAID1:
+        *raid1
+    - ext4_uefi-staging:
+        machine: uefi-virtio-vga
+        settings:
+          YAML_SCHEDULE: schedule/staging/%TEST%@uefi-staging.yaml
+          INSTALLONLY: ""
+    - minimal+base
+    sle-12-SP5-Server-DVD-Y-Staging-x86_64:
+    - autoyast_mini_no_product
+    - gnome
+    - cryptlvm_minimal_x
+    - default_install:
+        *default_install
+    - installcheck
+    - migration_zdup_offline_sle12sp2_64bit-staging:
+        *zdup
+    - rescue_system_sle11sp4
+    - RAID1:
+        *raid1
+    - ext4_uefi-staging:
+        *ext4
+    - minimal+base
+    ');
+    $t->post_ok(
+        "/api/v1/job_templates_scheduling/$job_group_id4",
+        form => {
+            schema   => $schema_filename,
+            template => YAML::XS::Dump($yaml)});
+    $t->status_is(200, 'New group with references was added to the database');
+    if (!$t->success) {
+        diag explain $t->tx->res->json;
+        return undef;
+    }
+    $t->get_ok(
+        "/api/v1/job_templates",
+        form => {
+            test_suite_name => 'RAID1',
+            flavor          => ''
+        })->status_is(200);
+    for my $json (@{$t->tx->res->json->{JobTemplates}}) {
+        say Mojo::Util::dumper $json->{settings};
```

Comment:
This should probably not be in the final test.

Reply:
oops

Reply:
fixed